### ISSUE TITLE
ReadyNAS 6.9.4 - Par: Success Unpack: Failure

### ISSUE THREAD

**Issue by Troutt025** [nzbget/nzbget#601]
So I just finished setting up NZBGet with CouchPotato and a few files downloaded successfully yesterday. Now today, none of them will unpack properly. I've tried multiple different files, and I also have a log saved. 
[iron man 3 2013 ac3 bdrip xvid gur part01 rar 03 63 iron man 3 2013 ac3 bdrip xvid gur part01 rar yenc 1 41.cp(tt1300854).log](https://github.com/nzbget/nzbget/files/2738713/iron.man.3.2013.ac3.bdrip.xvid.gur.part01.rar.03.63.iron.man.3.2013.ac3.bdrip.xvid.gur.part01.rar.yenc.1.41.cp.tt1300854.log)

Not really sure how to fix this.
Thanks for any help.

**Comment by hugbug** [comment]
The log contains a message about not being able to create output directory. Wrong path or permissions probably.

BTW where did you get the package? I assume it's not from nzbget download page as it has wrong unrar as well (unrar complaints about invalid parameters). Why don't you use official nzbget installer from nzbget download page?

**Comment by Troutt025** [comment]
So I'm using the app that is currently available on ReadyNAS. Is that not the official nzbget app? Also, I didn't change any of the directories. All of the paths are kept on default.

**Comment by hugbug** [comment]
Official packages can be downloaded from https://nzbget.net/download. Everything else is built by thirdparties. I have no idea how good or bad the package you've installed is. It may be shipped with errorneuos default configuration, include non-compatible unrar etc.

From the log it seems to work mostly. If you fix path in DestDir option and find proper unrar (official unrar from rarlab.com) maybe you can get everything work.

If you want to try official version, here is the manual - https://nzbget.net/installation-on-linux. It should work directly after installation. You need terminal access to your device.

**Comment by Troutt025** [comment]
![failed2](https://user-images.githubusercontent.com/25396471/50940436-987b6280-1446-11e9-8c23-8f361ce8d123.png)
Thanks for your reply, I'll try to download the official version today and tell you how that works out.

Okay, so I was able to install the latest NZBGet and start a download, now during the download I'm getting this error in the messages section. The download is still going like nothing is wrong, it's just not progressing.

![failed](https://user-images.githubusercontent.com/25396471/50937658-4c2a2580-143a-11e9-9710-d872b9bc67aa.png)

Fixed it by enabling encryption!


Everything seems to be fixed now!

To fix everything I just had to download the official NZBGet like you said and set it up properly.
Thanks for all the help!
